source/frontend/src/features/documents/documentDetail/DocumentDetailForm.tsx

```diff
@@ -105,13 +137,18 @@ export const DocumentDetailForm: React.FunctionComponent<
                         }
                         field={'documentTypeId'}
                         options={documentTypeOptions}
-                        onChange={props.onDocumentTypeChange}
+                        onChange={onDocumentTypeChange}
                         disabled={
                           documentTypeOptions.length === 1 ||
                           props.relationshipType === ApiGen_CodeTypes_DocumentRelationType.Templates
                         }
                       />
                     </SectionField>
+                    {exists(documentTypePurpose) && (
+                      <SectionField label={null}>
```

Comment:
why label={null} instead of just no declaration?

Reply:
thats the way we have right now. The label is required on the section field. I like having no label instead of a required null